### ISSUE TITLE
Error Messages when Launching 2.04

### ISSUE THREAD

**Issue by BelAirJeff** [Franiac/TwitchLeecher#710]
When launching Twitch Leecher 2.04, I get the following error messages:
![1](https://user-images.githubusercontent.com/83692575/154683735-24becdb4-504a-44fc-9b19-bcfbd216d24a.PNG)
![2](https://user-images.githubusercontent.com/83692575/154683746-17f5f1f3-3e60-4fa1-a7a5-bba0db6661aa.PNG)

And then the app is unusable and looks like this: 
![3](https://user-images.githubusercontent.com/83692575/154683896-d934e545-ed64-4ff6-8ebe-f8d76d7dc7f0.PNG)

**Comment by Franiac** [comment]
Can you please post the contents of those logfiles?

**Comment by BelAirJeff** [comment]
I'm sorry where would this file be located?

**Comment by Franiac** [comment]
just type `%appdata%\TwitchLeecher` in the Explorer address bar...

**Comment by Franiac** [comment]
Sorry... `%appdata%\Twitch Leecher`

**Comment by BelAirJeff** [comment]
When I opened the runtime.xml file with notepad, it was blank.

**Comment by Franiac** [comment]
Shit... ok... your solution is to just delete the file. It will be created again.

My problem is, I do not know why TL sometimes manages to write an empty file. We had this issue already a couple days ago. Can you remember anything like killing TL with task manager or somethin other unusual?

**Comment by BelAirJeff** [comment]
Yep, deleting that file solved the problem for me! 

I can't remember anything off the top of my head, but I started getting this message when 2.03 launched I think. I was still on 1.84 and got the message so decided to upgrade to 2.03. And then 2.03 didn't work... and redownloaded 1.84. Continued to have the error messages, but I was able to download my VODs.

Today saw that 2.04 was released and thought it would solve my error messages and then after installation got those error messages and thought I should share with you.

Thanks for being so speedy with the responses.  I really appreciate it!
